apps/backend/api/voterGroups/voterGroups.yaml

```
@@ -0,0 +1,134 @@
+summary: 'Create and read ballot voter groups'
+description: |
+  A voter group is always linked to one user and zero or several ballot papers.
+  You can create a new one ore read all the existing ones for an user.
```

This one of the cases I was recently talking about, where there is a vowel at the start of a word but still 'a' is used because the 'u' does not start with a "vowel-sound" when spoken.
```suggestion
  You can create a new one ore read all the existing ones for a user.
```

---

Fixed in a073463b50d353b079ab85e517e151574287af14